taskw/exceptions.py

```diff
@@ -18,4 +18,4 @@ def __unicode__(self):
         )
 
     def __str__(self):
-        return self.__unicode__().encode(sys.getdefaultencoding(), 'replace')
+        return self.__unicode__().encode(sys.getdefaultencoding(), 'replace').decode(sys.getdefaultencoding())
```

Comment:
Just briefly glancing, I’m fairly sure this is here from when we were supporting Python 2.7  where this method had to return bytes (& I believe we’ve dropped that by now).

Rather than calling `__unicode__`, encoding  to bytes, then decoding once again, we should really probably just move the contents of `__unicode__` here and delete the `__unicode__` method entirely.

Reply:
FYI - did this as part of https://github.com/ralphbean/taskw/pull/149